locales/en/translation.json

```diff
@@ -209,7 +209,8 @@
     "open-new-tab": "Open new tab",
     "open-verselist-in-new-tab": "Open verse list in new tab",
     "footnote-character": "n",
-    "adjust-font-size-tags-notes": "Adjust tags & notes font size"
+    "adjust-font-size-tags-notes": "Adjust tags & notes font size",
+    "new-note-hint": "Click to add a note"
```

Comment:
Please add the new text in all other locale files as well. This way we turn it into a todo for our translator friends.